test/hotspot/jtreg/vmTestbase/nsk/monitoring/share/thread/ThreadUtils.java

```diff
@@ -218,20 +228,33 @@ public static void threadDump(Log log, ThreadInfo[] tinfos) {
                 }
         }
 
+        // Most uses of waitThreadState usually succeed without retries.
+        // Sleep time increased to help avoid spurious failures.
         public final static int waitThreadStateRetries = 10;
-        public final static long waitThreadStateSleepTime = 100;
+        public final static long waitThreadStateSleepTime = 1000;
 
         public static void waitThreadState(Thread thread, Thread.State state) {
                 int retries = 0;
                 long ctime = System.currentTimeMillis();
+                long ctime2 = 0;
                 while (thread.getState() != state) {
-                        if (retries++ > waitThreadStateRetries)
-                                throw new TestFailure("Thread " + thread + " with current state " + thread.getState() + " did not reach state " + state + " with number of  retries: " + retries + ", time: " + (System.currentTimeMillis() - ctime));
+                        if (retries++ > waitThreadStateRetries) {
+                            // Failed to see desired state, give info to help diagnose the isuse.
```

Comment:
This file is using an indent of 8. Looks odd to use 4 for the modified code only. Maybe you could fix all the indents before integration?